### DISCUSSION TITLE
We plan to temporarily not provide a complete dashboard implementation. Any ideas?

### DISCUSSION THREAD

**Discussion by nextify2025** [nextify-limited/saasfly#5] [General]
We want to just provide a simple static dashfrontier data ，Because we want to provide integration with Headless CMS.

**Comment by dealmengor** [comment]
Hi @tianzx I can suggest you this:

- https://payloadcms.com/
- https://www.sanity.io/

**Reply by nextify2025** [reply]
> Hi @tianzx I can suggest you this:
> 
> * https://payloadcms.com/
> * https://www.sanity.io/

Thanks a lot for your suggestions! I really appreciate the input. Right now, I'm still thinking about the CMS part—I've got some ideas floating around, but nothing’s set in stone yet, and there might be changes down the line depending on how things shake out. For now, I’m leaning toward using fairly lightweight options like bknd or pocketbase since they seem to fit what I’m going for.
